reflectapi-demo/src/tests/enums.rs

```diff
@@ -135,3 +135,123 @@ pub enum Nums {
 fn test_enum_rename_num() {
     assert_snapshot!(Nums);
 }
+
+#[test]
+fn test_internally_tagged_enum_with_tuple_variants() {
+    #[derive(reflectapi::Input, reflectapi::Output, serde::Deserialize, serde::Serialize)]
+    struct Data {
+        a: String,
+        #[serde(skip_serializing_if = "Option::is_none")]
+        b: Option<u32>,
+    }
+
+    #[allow(dead_code)]
+    #[derive(reflectapi::Input, reflectapi::Output, serde::Deserialize, serde::Serialize)]
+    #[serde(tag = "type")]
+    enum E {
+        #[serde(rename = "a")]
+        A(Data),
+
+        #[serde(rename = "b")]
+        B { c: String },
+
+        #[serde(rename = "c")]
```

Comment:
This test doesn't seem as minimal as it could be. The only variant we care about here is `C` right?